doc/arch/adr-001-removing-smartdown.md

```diff
@@ -0,0 +1,48 @@
+# ADR 1: Removing Smartdown
+
+## Context
+
+[Smart Answers][smart-answers-github] have been historically painful to develop and maintain. In particular, relatively simple content changes can incur quite a high development cost.
+
+[Smartdown][smartdown-github] was developed in an attempt to reduce the overhead of Smart Answers development. [Smartdown was added to the Smart Answers project on 6 Aug 2014][smartdown-in-smart-answers].
+
+Smartdown made it easier for content designers to edit questions and outcome content, but was quite restrictive when it came to defining the rules of the Smart Answer (see the rules in [pay-leave-for-parents/partner_earned_more_than_lower_earnings_limit][spl-complicated-next-node-rules], for example).
+
+Smartdown didn't match Ruby Smart Answers in terms of features. Adding support for these features to Smartdown would have required extensions to the grammar and hence the parser. The cost of this would've been relatively high, especially considering that Ruby Smart Answers already had all the functionality we needed (apart from multiple questions-per-page).
```

Comment:
> Ruby Smart Answers already had all the functionality we needed

I think it would be worth qualifying that one of the benefits/functionality of Smartdown was the intent for it to be pased over an API, allowing publishing workflow. The decision here (which I :+1: with) is to not attempt to solve that problem, and focus on making the Ruby format as simple and accessible as possible, on the assumption of high turnover of people working on it.